test/fixtures/output/ip_route.out

```diff
@@ -1,3 +1,3 @@
-10.13.32.0/24 dev eth0  proto kernel  scope link  src 10.13.32.195 
-10.163.0.1 dev tun0  proto kernel  scope link  src 10.163.0.6 
-10.163.2.0/28 dev eth1  proto kernel  scope link  src 10.163.2.1 
+10.13.32.0/24 dev eth0  proto kernel  scope link  src 10.13.32.195  metric 600 
+10.163.0.1 dev tun0  proto kernel  scope link  src 10.163.0.6  metric 600 
+10.163.2.0/28 dev eth1  proto kernel  scope link  src 10.163.2.1  metric 600 
```

Comment:
I think this test case could be improved, I think the same metric everywhere is not realistic. FWIW, on my machine, with IPs censored:
```console
default via 9.9.9.9 dev eth0  proto static  metric 100 
default via 9.9.9.9 dev wlan0  proto static  metric 600 
9.9.9.9/18 dev eth0  proto kernel  scope link  src 9.9.9.9  metric 100 
9.9.9.9/23 dev wlan0  proto kernel  scope link  src 9.9.9.9  metric 600 
9.9.9.9 via 9.9.9.9 dev wlan0  proto dhcp  metric 600 
9.9.9.9/16 dev docker0  proto kernel  scope link  src 9.9.9.9 
9.9.9.9/24 dev bacbr  proto kernel  scope link  src 9.9.9.9 
9.9.9.9/24 dev virbr1  proto kernel  scope link  src 9.9.9.9 
9.9.9.9/24 dev virbr0  proto kernel  scope link  src 9.9.9.9 
```